src/storage/storage.cpp

```diff
@@ -445,10 +445,11 @@ void Storage::PopulateLayout(DataLayout &layout)
     {
         io::FileReader maneuver_overrides_file(config.GetPath(".osrm.maneuver_overrides"),
                                                io::FileReader::VerifyFingerprint);
-        const auto number_of_overrides = maneuver_overrides_file.ReadElementCount64();
+        const auto number_of_overrides =
```

Comment:
This is another bug fix: Without skipping the bytes of the vector this would read garbage data.